.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_KEY=1cb7RG1N3agHLuchuLs2GeqtKtdcyELRMj5HLHf8p48
```

Comment:
The .env should not be in git... you need to add to the `.gitignore` as well as do a `git rm --cached .env` to totally untrack it.

Reply:
removed it, it must have been a hold over from something.